### PR TITLE
Add tuning tips

### PR DESCRIPTION
**Pull request by JustinPihony**
This provides more details on how to decrease latency - especially for mostly idle systems.

### CODE REVIEW

File: docs/src/main/paradox/configuration.md [hunk 1]
```diff
@@ -145,3 +145,11 @@ akka-persistence-jdbc {
...
 The plugin automatically shuts down the HikariCP connection pool when the ActorSystem is terminated.
 This is done using @apidoc[ActorSystem.registerOnTermination](ActorSystem).
+
+## Tuning for Lower Latency
+
+The `jdbc-read-journal.journal-sequence-retrieval.query-delay` configuration controls how often the actor queries for new data. The default is `1s`, but this can be set lower for latency-sensitive applications to reduce the time between data retrievals.
+
+Similarly, `jdbc-read-journal.refresh-interval` dictates how often the system polls for new events when idle, also defaulting to `1s`. In mostly idle applications that still require low latencies, it is important to adjust both `query-delay` and `refresh-interval` to achieve optimal performance. Lowering just one of these values might not be sufficient for reducing latency.
+
+As with any performance tuning, it’s important to test these settings in your environment to find the right balance. Reducing these intervals will increase the load on your database, as each node in the cluster will be querying the read journal more frequently.
```
Comment (octonato) [inline docs/src/main/paradox/configuration.md:155]:
```suggestion
As with any performance tuning, it’s important to test these settings in your environment to find the right balance. Reducing these intervals will increase the load on your database, as each node in the cluster will be querying the journal more frequently.
```

Referring to it as read journal may give the impression that there is a read and a write journal. We only have a journal.

Comment (octonato) [inline docs/src/main/paradox/configuration.md:155]: Alternatively, we can say 'event journal'.

```suggestion
As with any performance tuning, it’s important to test these settings in your environment to find the right balance. Reducing these intervals will increase the load on your database, as each node in the cluster will be querying the event journal more frequently.
```

**Review by octonato — APPROVED** [review]
LGTM, with a minor phrasing suggestion

File: docs/src/main/paradox/configuration.md (hunk 1 above)
Comment (JustinPihony) [reply docs/src/main/paradox/configuration.md]: Hrmm, I agree - I accepted the first - but like event better - will use this one

**Review by johanandren — APPROVED** [review]

**Pull request merged**